curriculum/updated-curriculum/html/basic-html-04.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+  <body>
+    <h1>Hello World</h1>
+    <h2>CatPhotoApp</h2>
+    <p>Click here to view more cat photos.</p>
+  </body>
+</html>
+
+<!-- Commenting allows you to leave messages without affecting the output displayed to the end user. It also allows you to make code inactive. A comment in HTML starts with <!--, contians any number of lines of text, and ends with -->. Add a comment above the p element with the text: "TODO: Add link to cat photos". For example the comment <!-- TODO: Remove h1 -->contains the text "TODO: Remove h1". -->
```

Comment:
We really want to make these concise and let people discover for themselves (or infer from context). Rather than explain the purpose of comments or that they can be multiple lines, or that they make the code inside them inactive, we should demonstrate this through subsequent examples. So I recommend we make the text as simple as:

You can leave comments in your code like this: <!-- TODO: Remove h1 -->. Comments in HTML start with <!-- and end with -->. Add a comment above the p element with the text: "TODO: Add link to cat photos".

Reply:
👍